portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -32,6 +32,15 @@ definition {
 			value1 = "${velocityValueTotal}");
 	}
 
+	macro assertChartTotalValuePerformanceByStep {
+		var key_SLABreachedValue = "${slaBreachedValue}";
+
+		AssertTextEquals(
+			locator1 = "WorkflowMetrics#METRICS_SLA_BREACHED_VALUE",
+			rowName = "Review",
```

Comment:
Hi Michael,
Would you please also add rowName as a parameter for the macro? So, we would also be able to validate another steps (eg: Update) and not only Review.

Reply:
added!